HtmlTextView/src/main/java/org/sufficientlysecure/htmltextview/HtmlTagHandler.java

```diff
@@ -372,6 +381,10 @@ private static Object getLast(Editable text, Class kind) {
         }
     }
 
+    public void setBulletIndent(float px) {
```

Comment:
Rename to setBulletIndentPx(...) so it's consistent with the other method name.  Also add javadoc to this public method.